copier.yml

```diff
@@ -126,6 +126,18 @@ generate_requirements_txt:
     Generate requirements.txt from addons manifests and optional overrides in setup.py
     files.
 
+use_ruff:
+  default: no
+  type: bool
+  help: Use ruff and ruff-format instead of flake8, autoflake, pyupgrade, isort, black.
+  when: "{{ odoo_version >= 14.0 }}"
+
+additional_ruff_rules:
+  type: yaml
+  default: []
+  help: List of additional ruff rules to enable.
+  when: "{{ use_ruff }}"
+
```

Comment:
I think you don't need to add this here.

Copier supports smart updates. Thus, if you need extra rules, all you have to do is edit the ruff config file. Changes will be preserved.

```suggestion
```

Reply:
In general I find that brittle with conflicts on updates. When we have question it is more robust.

Reply:
OK, nevermind. There are more than 1 way to do things right. 😉

Reply:
And other argument in favor of the question approach, is that we pick-up answers from the previous branches when creating new ones.